app/controllers/api/v1/keywords_controller.rb

```diff
@@ -0,0 +1,21 @@
+# frozen_string_literal: true
+
+module Api
+  module V1
+    class KeywordsController < ApplicationController
```

Comment:
How about defining a new base controller for the API:

```suggestion
    class KeywordsController < API::ApplicationController
```

Then `escue Pagy::OverflowError => error` could be defined at that base controller for all endpoint 💡 

See https://github.com/nimblehq/the1-api/blob/development/app/controllers/v2/application_controller.rb#L16 fr examples on how to handle application-wide errors

Reply:
Fixed in 38a90735bf1cac906131759d368498838ff8cfbe